src/test/java/com/wikia/webdriver/elements/mercury/components/discussions/common/PostEntity.java

```diff
@@ -35,6 +35,10 @@ public boolean hasTopNote() {
     return null != findTopNoteElement();
   }
 
+  public boolean hasImage() {
+    return !post.findElements(By.cssSelector(".post-image-inner-image")).isEmpty();
```

Comment:
I'd move By to very top so that everyone who wants to change selectors will not need to go through entire class